library/build.gradle

```diff
@@ -69,9 +70,9 @@ uploadArchives {
     repositories.mavenDeployer {
         beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
 
-        repository(url: sonatypeRepo) {
```

Comment:
I ran into the same issue. I end up keeping this in `git stash` and undoing it when I push a PR. Would be great if the build file were updated to allow others to build without needing to do that.